src/client.coffee

```diff
@@ -107,7 +107,7 @@ class Client
 
   notify: (err) ->
     defContext = {
-      language: 'JavaScript',
+      language: 'JavaScript'
```

Comment:
The convention is to drop commas from dictionary definition (at least we do this in ng).